components/admin_console/system_users/system_users.tsx

```diff
@@ -235,7 +243,7 @@ export default class SystemUsers extends React.PureComponent {
         this.setState({loading: false});
     }
 
-    getUserByTokenOrId = async (id) => {
+    getUserByTokenOrId = async (id: any) => {
```

Comment:
This should be `string`

Reply:
Thanks @jespino - have fixed this in the latest commit.